docs/marionette.view.md

```diff
@@ -3,7 +3,13 @@ not be accurate or up-to-date_**
 
 # Marionette.View
 
-A `View` is a view that represents an item to be displayed with a template.
This is typically a `Backbone.Model`, `Backbone.Collection`, or nothing at all.

Views are also used to build up your application hierarchy - you can easily nest
multiple views through the `regions` attribute.

**_Note: From Marionette v3.x, `Marionette.View` replaces
+A `View` is a view that represents an item to be displayed with a template.
```

Comment:
> A `View` represents a DOM to be display with a template.
The `View` gets rendered to display data from a `Backbone.Model`, `Backbone.Collection`, or neither.

What follows is nice and informative.